web/satellite/src/api/auth.ts

```diff
@@ -172,7 +172,7 @@ export class AuthHttpApi {
 
         const response = await this.http.post(path, JSON.stringify(body), false);
         if (!response.ok) {
-            throw new Error('can not register user');
+            throw new Error('we are unable to create your account. This is an invite-only alpha, please join our waitlist to receive an invitation');
```

Comment:
this error should be taken from server response.
Because there are 2 possible errors, internal server error and absence of registration token

Reply:
you can hardcode them here, but we should differentiate them by http status code at least